tailwind.config.js

```diff
@@ -15,14 +15,15 @@ module.exports = {
   theme: {
     extend: {
       fontFamily: {
-        sans: ['Inter var', ...defaultTheme.fontFamily.sans],
+        sans: ['Poppins', ...defaultTheme.fontFamily.sans],
         system: defaultTheme.fontFamily.sans,
-        hero: ['Comfortaa', 'Inter var', ...defaultTheme.fontFamily.sans],
+        hero: ['Poppins', 'Poppins', ...defaultTheme.fontFamily.sans],
```

Comment:
Since we are no longer using `Inter var` and `Comfortaa`, I think we can delete their font files under `src/fonts`